bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/js/summernote_init.js

```diff
@@ -0,0 +1,34 @@
+$(function() {
+	var summernoteconfig = ${summernoteconfig};
+	var summernoteconfigdefault = {
+		onImageUpload : function(files) {
+			$(files).each(function(){
+				var file = this;
+				var data = new FormData();
```

Comment:
is there any other way to transfer data without using iframe - I dislike that hack. :tired_face: